SPECS/fontconfig/fontconfig.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        library for configuring and customizing font access.
 Name:           fontconfig
-Version:        2.13.91
+Version:        2.13.95
 Release:        1%{?dist}
 License:        BSD/GPL
```

Comment:
Having trouble verifying the license for this one,

can someone make sense of the COPYING.md here? 
  https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/main/COPYING

Reply:
@christopherco any thoughts here?

Reply:
Hmm, looks to be a few licenses together.
- MIT overall
- fc-case/CaseFolding.txt = Unicode
- src/fcatomic.h = MIT
- src/fcfoundry.h = MIT
- src/fcmd5.h = Public Domain
- src/fcmutex.h = MIT
- src/ftglue.[ch] = Public Domain